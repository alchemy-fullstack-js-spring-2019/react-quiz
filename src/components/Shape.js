import React from 'react';
import PropTypes from 'prop-types';


function Shape({ text, textColor, backgroundColor}) {

 //returns a div 
 
}

Shape.propTypes = {
  text: PropTypes.string.isRequired,
  textColor: PropTypes.string.isRequired,
  backgroundColor: PropTypes.string.isRequired
};

export default Shape;
