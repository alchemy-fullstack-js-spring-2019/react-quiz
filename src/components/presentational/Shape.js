import React from 'react';
import PropTypes from 'prop-types';

export default function Shape({ text, backgroundColor, textColor }) {
  return (
    <div style={{ width: '100px', height: '100px', backgroundColor: backgroundColor, color: textColor }}>{text}</div>
  );
}

Shape.propTypes = {
  backgroundColor: PropTypes.string, 
  textColor: PropTypes.string, 
  text: PropTypes.string 
};
