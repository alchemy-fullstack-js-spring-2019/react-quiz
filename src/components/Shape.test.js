import React from 'react';
import { shallow } from 'enzyme';
import Shape from './Shape';

describe('TextSelector component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Shape color="" backgroundColor="" text="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
