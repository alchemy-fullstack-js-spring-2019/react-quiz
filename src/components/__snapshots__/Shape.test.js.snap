// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextSelector component matches snapshot 1`] = `
<div
  style={
    Object {
      "background": "",
      "border": "2px solid black",
      "height": "200px",
      "width": "400px",
    }
  }
>
  <p
    style={
      Object {
        "color": "",
      }
    }
  />
</div>
`;
